fast64_internal/panels.py

```diff
@@ -1,40 +1,32 @@
 import bpy
 
-sm64GoalImport = "Import"  # Not in enum, separate UI option
-sm64GoalTypeEnum = [
-    ("All", "All", "All"),
-    ("Export Object/Actor/Anim", "Export Object/Actor/Anim", "Export Object/Actor/Anim"),
-    ("Export Level", "Export Level", "Export Level"),
-    ("Export Displaylist", "Export Displaylist", "Export Displaylist"),
-    ("Export UI Image", "Export UI Image", "Export UI Image"),
-]
-
 
 class SM64_Panel(bpy.types.Panel):
     bl_space_type = "VIEW_3D"
     bl_region_type = "UI"
     bl_category = "SM64"
     bl_options = {"DEFAULT_CLOSED"}
-    # goal refers to the selected sm64GoalTypeEnum, a different selection than this goal will filter this panel out
+    # goal refers to the selected enum_sm64_goal_type, a different selection than this goal will filter this panel out
+    # "Import" is not in the enum, as it is a separate UI option
     goal = None
```

Comment:
the comment makes it sound like `goal` can't be "Import" but it can

Reply:
Honestly I will just add an import bool instead, since you probably don´t want tons of import stuff when you have your goal set for level